odo/convert.py

```diff
@@ -206,6 +206,15 @@ def _():
 @convert.register(chunks(pd.DataFrame), Iterator, cost=10.0)
 def iterator_to_DataFrame_chunks(seq, chunksize=1024, **kwargs):
     seq2 = partition_all(chunksize, seq)
+
+    try:
+        if kwargs['add_index'] is True:
```

Comment:
we should just check the truthiness of the value instead of doing pointer comparisons against 'true'.

Also, to remove the try/except we can just use `dict.get` like: `if kwargs.get('add_index'):`. This function does does a lookup and returns some value if the element is not in the dict, defaulting to None (which acts as false)